src/imports.jl

```diff
@@ -34,7 +34,7 @@ import .DataGraphs:
 
 import Graphs: SimpleGraph, is_directed, weights
 
-import KrylovKit: eigsolve, linsolve
+import KrylovKit: eigsolve, linsolve, exponentiate
```

Comment:
I don't see why `exponentiate` needs to get imported since from what I can see it isn't being overloaded.